index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
Can we add the emmet generated HTML head tag boilerplate? It'll generate the meta tags for you which further improves accessibility. You can do this by typing '!' at the top of the file and pressing enter or tab.